projects/js-packages/components/components/icons/style.module.scss

```diff
@@ -13,6 +13,7 @@
 	fill: $gray-700;
 	&.facebook {
 		fill: var( --color-facebook );
+		border-radius: 50% !important;
```

Comment:
For some reason the colours for the tumblr logo on this branch. I'm not sure what's causing that:

<img width="39" alt="image" src="https://github.com/Automattic/jetpack/assets/96462/b892158f-7481-40de-a83f-26ee7926965b">
This branch

<img width="39" alt="image" src="https://github.com/Automattic/jetpack/assets/96462/78391d0b-aa1c-4c19-827a-e6314c53c1b0">
Trunk

It actually looks like it's using a different logo somehow.

I haven't got a twitter connection anymore but the colours were inverted there too